code/datums/emergency_calls/unitologi.dm

```diff
@@ -0,0 +1,44 @@
+/datum/emergency_call/unitologiststeam
+	name = "Unitologist"
+	probability = 25
+	landmark_tag = "unitologiststeam"
+	specials_outfits = list(/decl/hierarchy/outfit/healer, /decl/hierarchy/outfit/mechanic, /decl/hierarchy/outfit/faithful)
+
+/datum/emergency_call/unitologiststeam/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You are Unitologist.</b>")
+	to_chat(H, "")
+	to_chat(H, "<B>Follow any orders directly from Unitologist Central Command.</b>")
+
+/datum/emergency_call/unitologiststeam/create_member(datum/mind/M)
+	. = ..()
+	if(!.)
+		return
+
+	var/mob/original = M.current
+	var/mob/living/carbon/human/H = .
+
+	H.name = pick(GLOB.first_names_female + GLOB.first_names_male) + " " + pick(GLOB.last_names)
```

Comment:
forcibly giving people a random name is odd, do they have a way to change it?

Reply:
We can give them the opportunity to change the Name.